components/datetime/src/format.rs

```diff
@@ -25,10 +25,10 @@ use writeable::Writeable;
 /// # use icu_datetime::{DateTimeFormat, DateTimeFormatOptions};
 /// # use icu_datetime::date::MockDateTime;
 /// # use icu_provider::inv::InvariantDataProvider;
-/// # let locale = langid!("en").into();
+/// # let langid = langid!("en");
```

Comment:
Here and elsewhere: I'd like to keep the examples using `Locale` rather than `LanguageIdentifier` because people should not generally be using `LanguageIdentifier`.

```rust
let locale: Locale = langid!("en").into();
```